x-pack/legacy/plugins/apm/server/lib/settings/apm_indices/get_apm_indices.ts

```diff
@@ -54,15 +54,25 @@ export function getApmIndicesConfig(config: APMConfig): ApmIndicesConfig {
   };
 }
 
-export async function getApmIndices(context: APMRequestHandlerContext) {
+// export async function getApmIndices(context: APMRequestHandlerContext) {
+//   return _getApmIndices(context.core, context.config);
+// }
```

Comment:
leftover?

Reply:
I'll clean this up on the followup branch we have 👍